### PR TITLE
Package as RPM

### PR DESCRIPTION
**Pull request by jgrund**
package view-server as an RPM.
Try out using Fake to do this.

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Target.create "NpmBuild" (fun _ ->
+  Fake.JavaScript.Npm.install(id)
+  Fake.JavaScript.Npm.run "postversion" id
+  Fake.JavaScript.Npm.exec ("pack " + pwd) (fun o -> { o with WorkingDirectory = sources })
+)
+
+Target.create "BuildSpec" (fun _ ->
+  let v =
+    Fake.IO.File.readAsString "package.json"
```
Comment (johnsonw) [inline build.fsx]: couldn't you use `getPackageVersion` here?

File: iml-view-server.spec.template [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿%define base_name view-server
+%define base_prefixed iml-%{base_name}
+
+Name:       %{base_name}
```
Comment (johnsonw) [inline iml-view-server.spec.template]: Just wondering why `iml-view-server` isn't the full name?

File: iml-view-server.spec.template [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+mkdir -p %{buildroot}%{nodejs_sitelib}/@iml/%{name}
+
+%clean
+rm -rf %{buildroot}
+
+%files
+%{nodejs_sitelib}
+
+%changelog
+* Thu Mar 15 2018 Brian J. Murrell <brian.murrell@intel.com> - 6.3.0-1
```
Comment (johnsonw) [inline iml-view-server.spec.template]: Is this correct?

File: build.fsx (hunk 1 above)
Comment (jgrund) [reply build.fsx]: yes, this isn't ready for review yet, just a code dump at the moment.

File: iml-view-server.spec.template (hunk 3 above)
Comment (jgrund) [reply iml-view-server.spec.template]: Nope.

File: build.fsx (hunk 1 above)
Comment (johnsonw) [reply build.fsx]: 👍

File: iml-view-server.spec.template (hunk 2 above)
Comment (jgrund) [reply iml-view-server.spec.template]: No reason I can think of for it not to be.

**Pull request merged**